### PR TITLE
Fix static initialization order bug affecting curses console

### PR DESCRIPTION
**Pull request by slipher**

### CODE REVIEW

File: daemon/src/engine/sys/con_curses.cpp [hunk 1]
```diff
@@ -66,7 +66,9 @@ static bool     forceRedraw = false;
 static int      stderr_fd;
 #endif
 
-static const int LOG_LINES = ( LINES - 3 );
+static int NumLogLines() {
```
Comment (DolceTriade) [inline daemon/src/engine/sys/con_curses.cpp]: inline?

**Comment by DolceTriade** [comment]
Could you also expand on the bug a little?

File: daemon/src/engine/sys/con_curses.cpp (hunk 1 above)
Comment (slipher) [inline daemon/src/engine/sys/con_curses.cpp]: I'm sure it will be inlined anyway. Everyone says the inline keyword is an anachronism.

**Comment by slipher** [comment]
None of the text is displayed in the curses console.

**Comment by DolceTriade** [comment]
I meant in the commit message and the reason behind the initialization order bug. Like, "NUM_LINES isn't initialized in time because so and so"

**Comment by slipher** [comment]
All right, I added some more descriptions.

**Comment by DolceTriade** [comment]
Sweet thanks!

**Pull request merged**